ansible-ipi-install/roles/installer/tasks/25_customize_filesystem.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Create OpenShift Ignition Configs
+  shell: |
+    /usr/local/bin/openshift-baremetal-install --dir {{ dir }} create ignition-configs
+
+- name: Get Filetranspiler
+  get_url:
+    url: https://raw.githubusercontent.com/ashcrow/filetranspiler/master/filetranspile
```

Comment:
What will happen in a disconnected env?

Reply:
This is a good point. I suppose it would need to be solved with documentation.

Reply:
Chatting w/ @rlopez133 it sounds like disconnected needs to be its own feature effort.